www/resources/views/pages/consolelog.blade.php

```diff
@@ -0,0 +1,89 @@
+@extends('default')
+
+@section('style')
+<style>
+    #console-log {
+        text-align: left;
+        width: 100%;
+        display: block;
+        margin-left: auto;
+        margin-right: auto;
+    }
+
+    #console-log th {
+        padding: 0.2em 1em;
+        text-align: left;
+    }
+
+    #console-log td {
+        padding: 0.2em 1em;
+        max-width: 100px;
+    }
+
+    #console-log .message {
+        max-width: 700px;
+    }
+
+    #console-log .url {
+        max-width: 300px;
+    }
+
+    div.scrollable {
+        max-width: 100em;
+        margin: 0;
+        padding: 0;
+        overflow: auto;
+    }
+
+    tr.even {
+        background: whitesmoke;
```

Comment:
how do you know all these color names??

Reply:
haha, I don't, unfortunately. They do seem handy :) 
The css+html is a copy of the old code, that's currently not reachable: https://github.com/WPO-Foundation/webpagetest/blob/master/www/screen_shot.php#L88
So the purpose of the diff is two-fold: to bring back the feature and while Blade-ing